consent-ui/src/app/ais/initiation/common/dedicated-access/dedicated-access.component.html

```diff
@@ -0,0 +1,16 @@
+<form [formGroup]="limitedAccountAccessForm">
+  <div class="card">
+    <div class="card-body">
+
+      <h2 class="mb-4 text-left">Dedicated access for accounts in {{aspspName}}</h2>
+
+      <p class="text-secondary mb-5 text-left">Accounts with their details to be visible for <strong>{{finTechName}}</strong></p>
```

Comment:
Its better to avoid styling with html tags

Reply:
What's the reason for it - I want to stress those letters. If not - what is the alternative?

Reply:
as an alternative: <p class="font-weight-bold">Bold text.</p>